vcpkg/consume/snippets/git-registries/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.18)
```

Comment:
Can we put these in a directory or something to which we can link users for the complete example? (Or is there already one on that NorthWindTraders accouhnt?)